### DISCUSSION TITLE
Future of this extension

### DISCUSSION THREAD

**Discussion by michalmiddleton** [tomaciazek/vscode-ansible#33] [General]
Hi Tomasz, 

I am curious what are your plans with this project going forward. RedHat seems to have reused parts of your work and released their own Ansible language support extension.

Thank you,
Michal

**Comment by tomaciazek** [comment]
Hi Michal,
Sorry for late response, I must have missed the notification about this discussion.

To clarify, RedHat hasn't taken just parts of my work, but all of it, and I've helped them in the process. I've teamed up with RedHat to deliver a plugin with better support and assurances than I could provide myself. And because I couldn't really compete with such huge corporation 😉, and because such competition wouldn't really benefit the users.

The RedHat branded plugin has recently reached maturity and somewhat regular release cadence. The plan is to smoothly migrate users from my version of the plugin to RedHat's. There are just some technicalities to resolve and assure that the transition actually is as smooth as possible.

Cheers,
Tomasz
